src/ports/postgres/modules/kmeans/kmeans.py_in

```diff
@@ -34,6 +37,25 @@ def kmeans_validate_src(schema_madlib, rel_source, **kwargs):
 
 # ----------------------------------------------------------------------
 
+def kmeans_validate_expr(schema_madlib, rel_source, expr_point, **kwargs):
+    if not columns_exist_in_table(rel_source, [expr_point]):
+
+        p = re.compile('[Aa][Rr][Rr][Aa][Yy]\s*\[\s*[\s*\w|,\s*]+\s*\]')
```

Comment:
A comment on what the regex is doing would be very helpful. The array part is pretty obvious (though, couldn't that just be done with an insensitive regex??), but the rest is more difficult to follow.